### DISCUSSION TITLE
Creating a menu to choose an effect

### DISCUSSION THREAD

**Discussion by blaisebarrette** [HomeSpan/HomeSpan#504] [Q&A]
Hi all!

I have in my home a Raspberry Pi running the HomeBridge server. On there I have the Simple WLED plug-in that allows me to control a few esp32 running WLED from the home app.

this plug-in creates a color picker, and a menu allowing you to choose the effect you want the apply on your LED strip.

here is a screen shot:

![F152B1CE-DEA7-407B-A576-3F4CC4F15A70](https://user-images.githubusercontent.com/53702710/218008045-ab78555d-5609-482c-93d1-a7c8a7e55b80.png)

Is it possible to creat such menu in Homespan?

Thanks!

- Blaise

**Comment by HomeSpan** [comment]
See #458

**Reply by blaisebarrette** [reply]
Yeah! Thanks!!!
